.github/workflows/macos-check.yaml

```diff
@@ -0,0 +1,149 @@
+name: macOS Check
+on:
+  workflow_dispatch: # Manual trigger
+  pull_request:
+    paths-ignore:
+      - .gitignore
+      - .github/**
+      - '!.github/workflows/macos.yaml'  # Run check on self change
+      - CONTRIBUTORS
+      - LICENSE
+      - NOTICE
+      - README.md
+      - android/**
+      - iphone/**
+      - data/strings/**
+      - docs/**
+      - packaging/**
+      - pyhelpers/**
+      - tools/**
+      - '!tools/python/run_desktop_tests.py'
+      - '!tools/python/testserver.py'
+      - '!tools/python/SiblingKiller.py'
+      - xcode/**
+
+jobs:
+  macos-no-unity:
```

Comment:
This job is not needed, it will slow down the checks' finish time. Linux with no-unity is enough.

Reply:
Done